benchmarks/bench.py

```diff
@@ -0,0 +1,24 @@
+import apache_beam as beam
+from beam_sink import WriteToMySQL, ReadMySQL, MySQLConfig
+import os
+import json
+
+HOST = "localhost" if os.getenv("DEVELOP", False) else "mysql"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [os.getenv default type is builtins.bool. Expected str or None.](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5106141)